meta/group-info/th.yaml

```
@@ -0,0 +1,63 @@
+separator: ", "
+alumni:
+  roles:
+    academy: ผู้ค้นพบ osu!academy
+    admin: ผู้บริหารจัดการ
+    chat: ผู้ดูแลแชท
+    dev: ผู้พัฒนา osu!
+    media: ผู้สร้างสื่อ osu!
+    support: ทีมช่วยเหลือ
+    tournaments: ทัวร์นาเมนต์
+gmt:
+  all_mods: ผู้ดูแลทั้งหมด
+  areas:
+    beatmap: ผู้ดูแลบีทแมพ
+    chat: ผู้ดูแลการแชท
```

```suggestion
    chat: ผู้ดูแลแชท
```
Imo it doesnt need the word "การ"